x-pack/plugins/security_solution/scripts/endpoint/action_responder/utils.ts

```diff
@@ -114,3 +117,16 @@ export const sendEndpointActionResponse = async (
 
   return endpointResponse;
 };
+
+const getOutputDataIfNeeded = (command: EndpointActionData['command']) => {
```

Comment:
Please add a return type to the method.

Reply:
https://github.com/elastic/kibana/pull/134907/commits/ceb149cf2fc8decb7612d6aa00743dc31727c18e